### PR TITLE
Experience

### PR DESCRIPTION
**Pull request by blowymind99**

### CODE REVIEW

**Comment by NicolaePaulGeambazu** [comment]
good job

**Review by andreimaff — APPROVED** [review]
💯  💯  💯

File: src/_component/config/reusable.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+%rounder-border-list {
+    border-radius: 6px;
+    border:2px solid #707070;
```
Comment (robertoalx) [inline src/_component/config/reusable.scss:3]: Line 3 and line 8 have a missing space between ":" and "2px", it's all about consistency.

**Review by robertoalx — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]
Looking goooooood! 🚀 🤘

**Review by BogdanNegura — APPROVED** [review]
👍

File: src/_component/experience/experience.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+.experience__chevron,
+.experience__logo { 
+    grid-row: 1/3;
+}
+
+.experience__jobTitle { 
+    grid-column: 2/3;
+    grid-row: 1/2;
+}
```
Comment (RalucaRbc) [inline src/_component/experience/experience.scss:26]: You may need a new empty line between your 2 classes 😄

**Review by RalucaRbc — APPROVED** [review]
Looks almost perfect, well done 🤗

**Review by biggicode — APPROVED** [review]

File: src/_component/config/reusable.scss (hunk 1 above)
Comment (blowymind99) [reply src/_component/config/reusable.scss:3]: Your right! Thank you for your review!

**Pull request merged**